.github/workflows/build.yml

```diff
@@ -33,16 +33,17 @@ jobs:
         key: ${{ runner.os }}-${{ hashFiles('**/pom.xml') }}
         restore-keys: |
           ${{ runner.os }}-
-    - name: Set up Ruby 2.7
+    - name: Set up Ruby 3.3
       uses: ruby/setup-ruby@v1
       with:
-        ruby-version: 2.7
+        ruby-version: 3.3
     - name: Setup Environment
       shell: bash
       run: |
         echo "LANG=en_US.UTF-8" >> $GITHUB_ENV
         echo "MAVEN_OPTS=-Daether.connector.http.connectionMaxTtl=180 -DautoReleaseAfterClose=true" >> $GITHUB_ENV
-        echo "PMD_CI_SCRIPTS_URL=https://raw.githubusercontent.com/pmd/build-tools/master/scripts" >> $GITHUB_ENV
+        #echo "PMD_CI_SCRIPTS_URL=https://raw.githubusercontent.com/pmd/build-tools/master/scripts" >> $GITHUB_ENV
+        echo "PMD_CI_SCRIPTS_URL=https://raw.githubusercontent.com/adangel/build-tools/ruby3/scripts" >> $GITHUB_ENV
```

Comment:
is this needed? the changes as far as I can see are only those done to `check-environment.sh` which are already part of this PR…

Reply:
Yes, so that the CI build for this PR works - the script will actually download the `check-environment.sh` from this URL.
I'll undo this, when I merge it - which in fact, I'll do now.

Btw. this is the same change in the other PRs - this is only needed, that the CI-build for the PRs is working. On the master branches, this is not necessary anymore (given that this PR is merged).